webapp/src/components/cardDetail/cardDetail.tsx

```diff
@@ -103,6 +107,18 @@ const CardDetail = (props: Props): JSX.Element|null => {
         return null
     }
 
+    const allBlock = [] as ContentBlock[]
+    props.contents.forEach((block: ContentBlock|ContentBlock[]) => {
+        if (Array.isArray(block)) {
+            allBlock.push(...block)
+        } else {
+            allBlock.push(block)
+        }
+    })
+
+    const blocksWithAttachment = allBlock.filter((block) => block.type === 'attachment')
+    const blocksWithoutAttachment = allBlock.filter((block) => block.type !== 'attachment')
```

Comment:
These should be separate returns from server like comments.

Reply:
Okay got it.